tests/data/various_modules_package/function_module.py

```diff
@@ -54,7 +54,8 @@ def params(
     optional: Optional[int],
     tuple_: tuple[int, str, bool],
     literal: Literal["Some String"],
-    any_: Any
+    any_: Any,
+    callable_none: Callable[[int, float], None] | None,
```

Comment:
```suggestion
    callable_none: Callable[[int, float], None],
```

The `| None` is part of another issue.

Reply:
I think the `| None` here was necessary for code coverage, because of the new code. It shouldn't pose a problem, since we merge this code with the other issue anyway, right?